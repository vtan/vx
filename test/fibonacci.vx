(
  (let a 1)
  (let b 0)
  (let i 90)
  (while i (
    (let tmp (+ a b))
    (set b a)
    (set a tmp)
    (set i (- i 1))
  ))
  (set b 0)
)
