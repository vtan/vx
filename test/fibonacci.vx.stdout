00000000: 0000 0000 0000 0000 0000 0000 0000 0000  ................
00000010: 4557 32c0 b3cf ab40                      EW2....@
